packages/pwa/app/commerce-api/einstein.js

```diff
@@ -6,7 +6,7 @@
  */
 import fetch from 'cross-fetch'
 import {getAppOrigin} from 'pwa-kit-react-sdk/utils/url'
-import {keysToCamel} from './utils'
+import {keysToCamel, createAgent} from './utils'
```

Comment:
The Einstein API should never be called server side.

If it is, that is separate problem and can be addressed outside of the scope of the ticket.

We shouldn't look to add connection reuse to this code.